Makefile

```
@@ -212,6 +212,7 @@ proto-mocks: protoc
 
 service-clients:
 	@printf $(COLOR) "Generate service clients..."
+	@go build -o cached-mockgen ./cmd/tools/mockgen
 	@go generate ./client/...
```

I think this target should run only rpcwrappers, not mockgen. This bothered me before, and I see there's a way to fix it:
```suggestion
	@go generate -run rpcwrappers ./client/...
```

---

Fixed, thanks!